docs_nnx/guides/filters_guide.ipynb

```diff
@@ -149,14 +154,11 @@
    "id": "87c06e39",
    "metadata": {},
    "source": [
-    "## The Filter DSL\n",
+    "## The `Filter` DSL\n",
     "\n",
-    "To avoid users having to create these functions, Flax NNX exposes a small DSL, formalized\n",
-    "as the `nnx.filterlib.Filter` type, which lets users pass types, booleans, ellipsis,\n",
-    "tuples/lists, etc, and converts them to the appropriate predicate internally.\n",
+    "To help users avoid having to create functions mentioned in the previous section, Flax NNX exposes a small domain specific language ([DSL](https://en.wikipedia.org/wiki/Domain-specific_language)), formalized as the [`nnx.filterlib.Filter`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/filterlib.html) type. The `Filter` DSL allows users to pass types, booleans, ellipsis, tuples/lists, etc, and converts them to the appropriate predicate internally.\n",
```

Comment:
> To help users avoid having to create functions mentioned in the previous section, Flax NNX exposes a small domain specific language ([DSL](https://en.wikipedia.org/wiki/Domain-specific_language)), formalized as the [`nnx.filterlib.Filter`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/filterlib.html) type.

This line is verbose. Why not:

Flax NNX exposes a small domain specific language ([DSL](https://en.wikipedia.org/wiki/Domain-specific_language)), formalized as the [`nnx.filterlib.Filter`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/filterlib.html) type. This means users don't have to create functions like in the previous section.

Reply:
OK.

```
Flax NNX exposes a small domain specific language ([DSL](https://en.wikipedia.org/wiki/Domain-specific_language)), formalized as the [`nnx.filterlib.Filter`](https://flax.readthedocs.io/en/latest/api_reference/flax.nnx/filterlib.html) type. This means users don't have to create functions like in the previous section.
```